sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1431,4 +1484,30 @@ static void validateTableExists(BigtableConfig config, PipelineOptions options)
       }
     }
   }
+
+  interface ResourceStatsSupplier {
+    ResourceLimiterStats getStats();
+  }
+
+  private static class ResourceStatsSupplierImpl implements ResourceStatsSupplier, Serializable {
+
+    private final BigtableConfig bigtableConfig;
+
+    ResourceStatsSupplierImpl(BigtableConfig bigtableConfig) {
+      this.bigtableConfig = bigtableConfig;
+    }
+
+    @Override
+    public ResourceLimiterStats getStats() {
```

Comment:
does this need to be public?

Reply:
Due to extending the interface that is inherently public, the method has to be too.